.github/CHECKPOINT_ISSUE_TEMPLATE.md

```diff
@@ -12,7 +12,7 @@ How to compute a new checkpoint for calibnet:
 1. Install `forest-cli`
```

Comment:
`forest-cli` is not needed anymore for this subcommand.

Reply:
Corrected.